# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1295081993 18000
#      Sat Jan 15 03:59:53 2011 -0500
# Node ID 28bae9cd7ea93d98b6fb39d183e3600f70f57864
# Parent  b0ef6f28e09aa4b2180a204b3ad70d7224320151
ov-fcn-handle.cc: use version instead of flops in test

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,12 @@
+2011-01-15  John W. Eaton  <jwe@octave.org>
+
+	* ov-fcn-handle.cc: Use version instead of flops in test.
+
 2011-01-14  Rik  <octave@nomad.inbox5.com>
 
 	* DLD-FUNCTIONS/svd.cc: Add Seealso references to svd.
 
 2011-01-14  Rik  <octave@nomad.inbox5.com>
 
 	* src/mappers.cc: Add Seealso links between sqrt, cbrt, nthroot
 	functions.
diff --git a/src/ov-fcn-handle.cc b/src/ov-fcn-handle.cc
--- a/src/ov-fcn-handle.cc
+++ b/src/ov-fcn-handle.cc
@@ -1251,17 +1251,17 @@ octave_fcn_handle::load_hdf5 (hid_t loc_
 #endif
 
 /* 
 
 %!test
 %! a = 2;
 %! f = @(x) a + x;
 %! g = @(x) 2 * x;
-%! hm = @flops;
+%! hm = @version;
 %! hdld = @svd;
 %! hbi = @log2;
 %! f2 = f;
 %! g2 = g;
 %! hm2 = hm;
 %! hdld2 = hdld;
 %! hbi2 = hbi;
 %! modes = {"-text", "-binary"};
